### DISCUSSION TITLE
Is it possible to apply presets to M2O lookups?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13069] [Archive]
or should i use `display_template`. The doc says:
>How items in this collection should be displayed when viewed relationally in the Admin App.

But when I set the `Display Template` field in the `Collection Setup` it has no effect.

**Comment by rijkvanzanten** [comment]
I ehh don't really understand the question. Should you use `display_template` instead of `Display Template`? Are you comparing the display template option in the collection vs the interface? 🤔
